src/docker-utils.coffee

```diff
@@ -204,8 +204,7 @@ do ->
 						docker.getImage(tag).removeAsync()
 						.then ->
 							console.log('Deleted image:', tag, image.Id, image.RepoTags)
-						.catch (err) ->
-							console.log('Error deleting image:', tag, image.Id, image.RepoTags, err)
+						.catch (err) -> # Ignore image delete error
```

Comment:
You can also use `_.noop` to be a bit more English-language, and reusing a single noop function helps the optimiser a bit as well